src/server/req_quejob.c

```diff
@@ -1959,6 +1969,7 @@ req_jobscript(struct batch_request *preq)
 	}
 	if (fds < 0) {
 		log_err(errno, "req_jobscript", msg_script_open);
+		delete_link(&pj->ji_alljobs);
 		req_reject(PBSE_INTERNAL, 0, preq);
```

Comment:
This behavior is consistent with rest of the code. We have the same on requests.c:4738, req_quejob.c:2152. What do you recommend?